### PR TITLE
Adds the Coldridge-Class Corvette

### PR DESCRIPTION
**Pull request by AtlasRules**
## About The Pull Request

Adds the Coldridge-class Corvette, a military corvette designed for patrolling the sector, with a good amount of armaments.

## Why It's Good For The Game

more shiptests is good shiptests

## Changelog
:cl:
add: Adds the Coldridge-class Corvette
/:cl:

### CODE REVIEW

**Comment by AtlasRules** [comment]
hey shitty bots can you like stop fucking lying thanks

**Comment by AtlasRules** [comment]
@MarkSuckerberg please approve the workflow

**Comment by AtlasRules** [comment]
TODO:
- [x] add some more flavour and fluff
- [x] sort out the fire alarm directions
- [ ] (potentially) fix bugs

**Comment by ZephyrTFA** [comment]
`_maps/shuttles/shiptest/coldridge.dmm:1410:error: unterminated text (expecting ")`

File: _maps/shuttles/shiptest/coldridge.dmm [hunk 1]
```diff
@@ -0,0 +1,2601 @@
...
+/turf/open/floor/pod/light,
+/area/ship/crew)
+"JH" = (
+/obj/machinery/airalarm/directional/west,
+/obj/effect/decal/cleanable/cobweb,
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/minor/bowler_or_that,
+/obj/effect/spawner/lootdrop/maintenance/three,
+/obj/item/clothing/mask/rat/raven{
+	desc = "A mask made of soft vinyl and latex, it gives you a feeling of dishonor.
```
Comment (ZephyrTFA) [inline _maps/shuttles/shiptest/coldridge.dmm]:
```suggestion
	desc = "A mask made of soft vinyl and latex, it gives you a feeling of dishonor."

```

**Comment by AtlasRules** [comment]
pain, suffering even

**Comment by AtlasRules** [comment]
can i get that removed? I can't edit this map until I get my actual PC fixed, my laptop doesn't like running SDMM, FDMM or even DMM

**Comment by AtlasRules** [comment]
closed I don’t like this map anymore maybe I’ll make a better one someday

**Pull request closed without merging**